src/cli/cli.cc

```diff
@@ -1910,7 +1935,13 @@ int main (const int argc, const char* argv[]) {
 
       StringStream sdkmanager;
       StringStream packages;
-      StringStream gradlew;
+      StringStream gradlew;      
+      String ndkVersion = "25.0.8775105";
```

Comment:
eventually we may support this in `socket.ini`

Reply:
Agree, all of these android strings need to be configurable.